docs/architecture/cloud-native/definition.md

```diff
@@ -68,96 +69,95 @@ Cloud native systems embrace microservices. Moving beyond the popular monolithic
+Beyond cloud compute infrastructure, cloud native systems depend upon many different ancillary resources to operate: Data stores, message brokers, monitoring, and identity services - to name a few.
 
-A backing service is any external resource on which an application relies for functionality. While they can be local or cloud-based, cloud native apps by definition always favor cloud-based services.
+A backing service is any external resource on which an application relies for functionality. Figure 1-7 shows many common backing services consumed by cloud native systems.
 
-Figure 1-7, shown below, depicts common backing services found in cloud native systems.
 
 ![Cloud native backing services](media/cloud-native-backing-services.png)
 **Figure 1-7**. Cloud native backing services
 
-Your operations team could accept the responsibility for owning these ancillary services. It wouldn’t be hard to provision a VM and install, say, an instance of the open source RabbitMQ message broker. While certainly feasible, does that make sense - especially from a business perspective?
+Your operations team could accept the responsibility for owning these ancillary services. It wouldn’t be hard to provision a VM and install an instance of the open source RabbitMQ message broker. While feasible, does that make sense - from a business perspective?
 
-Chances are you’re not in the message broker, database or identity server business. So, then, why incur the cost and complexity of owning these items when someone else can do it better and for less? No matter how skilled you may be, it is unlikely that you will be able to match the reliability and performance, say, of the fully-managed [Azure SQL Database](https://azure.microsoft.com/services/sql-database/) service in Azure. That Microsoft team that owns it lives and breathes databases, 24/7 - that’s their business. While you, on the other hand, sell consumer items, medical devices, etc. Do what you're good at and let others help you with the rest. Focus your effort on building outstanding customer functionality, not becoming experts in 3rd party/open-source backing services.
+Chances are you’re not in the message broker, database or identity server business. So, then, why incur the cost and complexity of owning these items when someone else can do it better and for less? No matter how skilled you may be, it is unlikely that you will be able to match the reliability and performance, say, of the fully-managed [Azure SQL Database](https://azure.microsoft.com/services/sql-database/) service in Azure. That Microsoft team that owns it lives and breathes databases - that’s their business. While you, on the other hand, sell consumer items, provide medical services, or deliver packages. Do what you're good at and let others help you with the rest. Focus your effort on building outstanding customer functionality, not becoming experts in open-source backing services.
```

Comment:
Adverbs ending in ly shouldn't have a hyphen

```suggestion
Chances are you’re not in the message broker, database, or identity server business. So, then, why incur the cost and complexity of owning these items when someone else can do it better and for less? No matter how skilled you may be, it's unlikely that you can match the reliability and performance of the fully managed [Azure SQL Database](https://azure.microsoft.com/services/sql-database/) service in Azure, for example. That Microsoft team that owns it lives and breathes databases - that’s their business. While you, on the other hand, sell consumer items, provide medical services, or deliver packages. Do what you're good at and let others help you with the rest. Focus your effort on building outstanding customer functionality, not becoming experts in open-source backing services.
```

Reply:
That Microsoft team that owns it lives and breathes databases -> This doesn't read well.

Perhaps
"The people working on the Microsoft team that owns [what? - that product?], they live and breathe databases all day"

Reply:
Perfect. Restated.